biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -45,6 +45,8 @@ module EDPatchDynamicsMod
   use FatesInterfaceTypesMod    , only : bc_in_type
   use FatesInterfaceTypesMod    , only : hlm_days_per_year
   use FatesInterfaceTypesMod    , only : numpft
+  use FatesInterfaceTypesMod    , only : hlm_use_nocomp
```

Comment:
@rosiealice , would we consider a system where the structural controls on demographics are passed in as an integer code, or a string name; as an alternative to more than 1 binary switches (eg hlm_use_nocomp and hlm_use_fixed_biogeog)

Reply:
punting this until the reduced complexity mode structure is more established....